src/compiler/scala/tools/nsc/settings/Warnings.scala

```diff
@@ -54,8 +54,10 @@ trait Warnings {
          |    The regex must match the full name of the warning position.
          |
          |  - Source file name: src=src_managed/.*
-         |    The regex must match the canonical path, starting at any path segment.
-         |    Example: `b/.*Test.scala` matches `/a/b/XTest.scala` but not `/ab/YTest.scala`.
+         |    If `-rootdir` is specified, the regex must match the canonical path relative to the
```

Comment:
We should also mention that file name separators are normalized to `/`

Reply:
3 lines below it says "Use unix-style paths, separated by `/`" - is that it, did you mean something else?

Reply:
Ugh, sorry, no idea how I missed that.